pygmt/src/grdimage.py

```diff
@@ -36,15 +44,16 @@ def grdimage(self, grid, **kwargs):
     Project and plot grids or images.
 
     Reads a 2-D grid file and produces a gray-shaded (or colored) map by
-    building a rectangular image and assigning pixels a gray-shade (or
-    color) based on the z-value and the CPT file. Optionally, illumination
-    may be added by providing a file with intensities in the (-1,+1) range
-    or instructions to derive intensities from the input data grid. Values
-    outside this range will be clipped. If GMT is built with GDAL support,
-    ``grid`` can be an image file (geo-referenced or not). In this case the
-    image can optionally be illuminated with the file provided via the
-    ``shading`` parameter. Here, if image has no coordinates then those of the
-    intensity file will be used.
+    building a rectangular image and assigning pixels a gray-shade (or color)
+    based on the z-value and the CPT file. Optionally, illumination may be
+    added by providing a file with intensities in the (-1,+1) range or
+    instructions to derive intensities from the input data grid. Values outside
+    this range will be clipped. Such intensity files can be created from the
+    grid using `grdgradient` and, optionally, modified by `grdmath` or
+    `grdhisteq`. If GMT is built with GDAL support, ``grid`` can be an image
```

Comment:
`grdgradient` has been wrapped in #1269 so we can point to the actual function now :grin:

```suggestion
    grid using :meth:`pygmt.grdgradient` and, optionally, modified by `grdmath`
    or `grdhisteq`. If GMT is built with GDAL support, ``grid`` can be an image
```